website/src/components/Errors/Codes/41.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export default function ErrorCode41(props) {
+  return (
+    <>
+      <h1>#41: single-spa is loaded on the page multiple times</h1>
+      <p>
+        The single-spa library should be loaded only one time on the page. This is important both for performance and to avoid bugs.
+        Any imported APIs 
+      </p>
+      <h2>To fix:</h2>
+      <p>
+        Mark single-spa as a <a href="https://webpack.js.org/configuration/externals/#root" rel="noopener">webpack external</a> in your root config and in each of your microfrontends.
```

Comment:
did you mean to add `target="_blank"` to this anchor?

Reply:
I was going back and forth on it after having read a criticism of target blank ha. But yes I'll add it, good catch.